### PR TITLE
Add sponsors section

### PR DESCRIPTION
**Pull request by dhcrain**
This will need some help with the content of the sponsors (esp Humanity Codes) 
relates to #9

### CODE REVIEW

File: src/config/sponsors.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  },
+  {
+    name: 'LEAP',
+    desc: 'This is a group for UX design practitioners and those interested in learning about design.',
+    logo: 'leap.png',
+    url: 'http://www.purelansing.com/'
+  },
+  {
+    name: 'Humanity Codes',
+    desc: '???',
```
Comment (egillespie) [inline src/config/sponsors.js]: Hmm, how about this?

> Humanity Codes uses code and education to bring together Lansing's talented and diverse tech community.

File: src/components/sponsors.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <resource-section
+    id="sponsors"
+    heading="Lansing Codes Sponsors"
```
Comment (egillespie) [inline src/components/sponsors.vue]: Would you mind using "Our Sponsors" here?

File: src/components/sponsors.vue [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    description="sponsors to help you become a better coder"
```
Comment (egillespie) [inline src/components/sponsors.vue]: How does this sound?

> our gracious sponsors make all of this possible!

File: src/config/sponsors.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    desc: 'Founded in 1987, TechSmith pioneered the revolutionary idea of capturing screen content for better communication. Today, TechSmith is the world’s #1 source for visual communication software.',
+    logo: 'tech_smith.png',
+    url: 'https://www.techsmith.com/'
+  },
+  {
+    name: 'Kunz, Leigh, and Associates',
+    desc: 'Kunz, Leigh and Associates is a software development and professional services company currently serving the private and public sectors in Michigan and Ohio.',
+    logo: 'KLA.png',
+    url: 'https://kunzleigh.com/'
+  }
```
Comment (egillespie) [inline src/config/sponsors.js]: Sorry I didn't catch this the first time. Would you mind adding Hyperverses as well? The link is on #9. @pfhawkins can provide a good image and description. 🙂

Comment (pfhawkins) [reply src/config/sponsors.js]: I have updated Hyperverses! This should be all set.

**Pull request merged**